xarray/core/dataarray.py

```diff
@@ -3440,18 +3441,13 @@ def quantile(
             Quantile to compute, which must be between 0 and 1 inclusive.
         dim : hashable or sequence of hashable, optional
             Dimension(s) over which to apply quantile.
-        interpolation : {"linear", "lower", "higher", "midpoint", "nearest"}, default: "linear"
```

Comment:
Shall we support `interpolation` for a little longer with a Deprecation Warning?